primitives/src/lib.rs

```diff
@@ -116,3 +107,15 @@ pub struct PriceWithDecimal {
     pub price: Price,
     pub decimal: u8,
 }
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
This comment format looks weird.. please check with rust codebase.

Reply:
it's more a divider to seperate types & traits